packages/ove-app-svg/README.md

```diff
@@ -0,0 +1,51 @@
+# SVG App
+
+This app supports visualisation of SVG using the OVE framework. It is based on [Tuoris](https://github.com/fvictor/tuoris), a middleware for distributed SVG visualisation.
```

Comment:
*rendering* or *display* of SVG *images*/*graphics*

Reply:
SVG == Scalable Vector `Graphics` so the second bit is redundant, but I will change to `rendering` instead of `visualisation of`, which sounds more accurate.